### ISSUE TITLE
Allow to suppress Permalink from preview/up output

### ISSUE THREAD

**Issue by omerlh** [pulumi/pulumi#4103]
Currently, when running preview/up commands the output contains `Permalink`, which (at least on AWS) contains sensitive information - which is an issue when posting the output to GitHub. I would like a flag that allows suppressing this link from the output.

**Comment by lukehoban** [comment]
> which (at least on AWS) contains sensitive information 

Could you clarify this?  Do you mean the URL itself does?  Or do you mean the page it links to (which should be authenticated) does?

> at least on AWS

Do you specifically mean when using the S3 state storage backend?

**Comment by omerlh** [comment]
It's a pre-signed AWS url to the bucket - valid for an hour. It's not horrible, but I don't want to show it there..

**Comment by lukehoban** [comment]
Got it - this is for the S3 state storage backend.

I think quite possibly we just shouldn't show the "permalink" at all for the filestate backends.

**Comment by glennsarti** [comment]
Also applicable to Azure Blob storage

```
Permalink: https://something.blob.core.windows.net/pulumi-state/.pulumi/stacks/dev.json?se=2020-06-17T08%3A17%...SAS KEY
```

**Comment by scott-the-programmer** [comment]
I'll pick this one up if that's OK. 

I'll try get a PR up soon :+1:

**Comment by lukehoban** [comment]
This was fixed with #5191.  There is now a `-suppress-permalink` option you can pass to the CLI.

**Comment by glennsarti** [comment]
🥳   Thanks @lukehoban !
